lib/bundlex/toolchain/common/unix/os_deps.ex

```diff
@@ -147,7 +147,7 @@ defmodule Bundlex.Toolchain.Common.Unix.OSDeps do
 
     # TODO: pass the platform via arguments
     # $ORIGIN must be escaped so that it's not treated as an ENV variable
-    rpath_root = if Bundlex.platform() == :macosx, do: "@loader_path", else: "\\$ORIGIN"
+    rpath_root = if Platform.get_target!() == :macosx, do: "@loader_path", else: "\\$ORIGIN"
```

Comment:
What is the difference between `Platform.get_target!()` and `Bundlex.get_target!()`?

Reply:
Now deprecated `Bundlex.get_platform/0` invoked `Platform.get_target!/0`, it gives a bit different information than `Bundlex.get_target/0`, we probably want to get rid of it eventually and rely just on the `Bundlex.get_target/0`

Reply:
So why won't you use `Bundlex.get_target/0` instead of `Platform.get_target!/0`?